workspace/src/main/openapi/workspace.yaml

```diff
@@ -486,6 +486,13 @@ components:
           description: Set this property to true to use a dedicated Azure Event Hub Namespace for this Workspace.
             The Event Hub Namespace must be named \'<organization_id\>-<workspace_id\>\' (in lower case).
             This Namespace must also contain two Event Hubs named \'probesmeasures\' and \'scenariorun\'.
+        sendScenarioMetadataToEventHub:
+          # Deprecated - this option will be removed in the future
+          type: boolean
+          default: false
```

Comment:
Set it to true by default? + useDedicatedEventHubNamespace to true too? => communicate when releasing on this default value change for existing workspace for people to force it to false if needed.

Reply:
sendScenarioMetadataToEventHub default value to true, ok
but don't understand for useDedicatedEventHubNamespace

Reply:
It is the property just before this one in workspace.yaml.
To get the new event system we need 2 options for the workspace set to true:
useDedicatedEventHubNamespace & sendScenarioMetadataToEventHub.

This 2 options were by default to false which means this new feature is not available by default.
* useDedicatedEventHubNamespace where set to false for backward compatibility but now it is a known feature and we should set it to true to always enable scenario metadata. For now solution must put explicitly useDedicatedEventHubNamespace to true if they want it, after this correction, Solutions that does not want a dedicated event hub will need to set it to false.
* you changed sendScenarioMetadataToEventHub to true by default to it's OK.

There is another way to handle this to think about: handle a platform compatibility version in Workspace. Depending on this version it will change this defaut values.